### DISCUSSION TITLE
Export to XLSX from v4?

### DISCUSSION THREAD

**Discussion by til** [activeadmin/activeadmin#8460] [Help]
Apparently there is a gem that adds XLSX download functionality to ActiveAdmin: https://github.com/caxlsx/activeadmin-caxlsx. Trying to install it in an ActiveAdmin v4 application causes a couple of errors, probably because it's outdated – so before I debug those, I wonder if someone has it working in a recent ActiveAdmin version already? Or maybe a different approach for XLSX export?

**Comment by tagliala** [comment]
Hello, I've participated to some development related to caxlsx

I can see that the activeadmin extension is still supporting Ruby 1.9 and uses `alias_method_chain`, removed long time ago from Rails

Is that still working even on v3 or v2?
